packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -57,16 +61,19 @@ export function DiscoverSection({
         </span>
       ),
     },
-    {
+  ];
+
+  if (searchValue !== SearchExperiment.V1) {
```

Comment:
Should we not say if it's explicitly control?
What happens if we introduce V2 for instance

Reply:
Makes sense. Let's do that.